server/src/test/java/org/elasticsearch/search/SearchServiceTests.java

```diff
@@ -2024,11 +2024,20 @@ public void testSupportsParallelCollection() {
         if (searchSourceBuilder != null && randomBoolean()) {
             searchSourceBuilder.aggregation(new TermsAggregationBuilder("terms"));
         }
-        assertTrue(SearchService.supportsParallelCollection(ResultsType.DFS, searchSourceBuilder));
+        assertTrue(SearchService.supportsParallelCollection(ResultsType.DFS, searchSourceBuilder, true));
         assertFalse(
             SearchService.supportsParallelCollection(
                 randomFrom(randomFrom(ResultsType.QUERY, ResultsType.NONE, ResultsType.FETCH)),
-                searchSourceBuilder
+                searchSourceBuilder,
+                false
+            )
+        );
+        assertFalse(
+            // TODO: until https://github.com/elastic/elasticsearch/pull/98425 is merged, this should always return false.
```

Comment:
I am updating the commented conditional, but not un-commenting it just yet. Assuming that I will merge my PR tomorrow my morning, you could uncomment it as part of your PR and update your test here accordingly.